### PR TITLE
fix, chore, refactor(egen): Adds numpy==1.23.0, removes version ==0.0.4 from vizier installation, replaces K80 with T4, refactors as per template guidelines

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following changes:

1. Adds numpy==1.23.0 installation and replaces google-vizier==0.0.4 to resolve dependency errors
2. changes REGION to LOCATION
3. Includes Colab restart Kernal and authentication as per template guidelines
4. Removes import os and IS_TESTING from clean up section
5. Adds code to delete locally generated file
6. Replaces K80 with T4
7. Adds collab enterprise logo
8. Makes structural changes according to the new notebook template
9. Removes the use of future tense
10. Contraction of words

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/vizier/get_started_vertex_vizier.ipynb [hunk 1]
```diff
@@ -64,7 +66,7 @@
         "\n",
         "This tutorial demonstrates how to use Vertex AI Vizier for hyperparameter tuning.\n",
         "\n",
-        "Learn more about [Vertex AI Vizier](https://cloud.google.com/vertex-ai/docs/vizier/overview)."
+        "Learn more about [Vertex AI Vizier overview](https://cloud.google.com/vertex-ai/docs/vizier/overview)."
```
Comment (kittyabs) [inline notebooks/official/vizier/get_started_vertex_vizier.ipynb:69]: remove "overview"

File: notebooks/official/vizier/get_started_vertex_vizier.ipynb [hunk 2]
```diff
@@ -75,13 +77,13 @@
...
-        "In this tutorial, you learn how to use `Vertex AI Vizier` for when training with `Vertex AI`.\n",
+        "In this tutorial, you learn how to use Vertex AI Vizier when training with Vertex AI.\n",
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Training`\n",
-        "- `Vertex AI Hyperparameter Tuning`\n",
-        "- `Vertex AI Vizier`\n",
+        "- Vertex AI Training\n",
+        "- Vertex AI Hyperparameter Tuning\n",
```
Comment (kittyabs) [inline notebooks/official/vizier/get_started_vertex_vizier.ipynb:85]: lower case: "hyperparameter tuning"

File: notebooks/official/vizier/get_started_vertex_vizier.ipynb [hunk 3]
```diff
@@ -833,7 +784,7 @@
       "source": [
         "### Prepare your disk specification\n",
         "\n",
-        "(optional) Now define the disk specification for your custom hyperparameter tuning job. This tells Vertex what type and size of disk to provision in each machine instance for the hyperparameter tuning.\n",
+        "(**optional**) Now define the disk specification for your custom hyperparameter tuning job. This tells Vertex about the type and size of disk to provision in each machine instance for the hyperparameter tuning.\n",
```
Comment (kittyabs) [inline notebooks/official/vizier/get_started_vertex_vizier.ipynb:787]: Add "AI" to "Vertex": "Vertex AI"

File: notebooks/official/vizier/get_started_vertex_vizier.ipynb [hunk 4]
```diff
@@ -861,32 +812,32 @@
...
-        "      - indirect: The service passes the Cloud Storage location as the environment variable `AIP_MODEL_DIR` to your training script (set variable `DIRECT = False`). In this case, you tell the service the model artifact location in the job specification.\n",
-        "  - `\"--epochs=\" + EPOCHS`: The number of epochs for training.\n",
-        "  - `\"--steps=\" + STEPS`: The number of steps (batches) per epoch.\n",
-        "  - `\"--distribute=\" + TRAIN_STRATEGY\"` : The hyperparameter tuning distribution strategy to use for single or distributed hyperparameter tuning.\n",
-        "     - `\"single\"`: single device.\n",
-        "     - `\"mirror\"`: all GPU devices on a single compute instance.\n",
-        "     - `\"multi\"`: all GPU devices on all compute instances."
+        "    - `args`: The command line arguments to pass to the corresponding Python module. In this example, you're going to configure\n",
+        "    \n",
+        "        - `--model-dir` : The Cloud Storage location where to store the model artifacts. There are two ways to tell the hyperparameter tuning script where to save the model artifacts:\n",
```
Comment (kittyabs) [inline notebooks/official/vizier/get_started_vertex_vizier.ipynb:830]: add "for": 
- "The Cloud Storage location for where to store the model artifacts."

File: notebooks/official/vizier/get_started_vertex_vizier.ipynb [hunk 5]
```diff
@@ -1146,7 +1097,7 @@
       "source": [
         "## Vertex AI Hyperparameter Tuning and Vertex AI Vizer service combined\n",
         "\n",
-        "The following example demonstrates how to setup, execute and evaluate trials using the Vertex AI Hyperparameter Tuning service with `Vizier` search service."
+        "The following example demonstrates how to setup, execute and evaluate trials using the Vertex AI Hyperparameter Tuning service with Vizier search service."
```
Comment (kittyabs) [inline notebooks/official/vizier/get_started_vertex_vizier.ipynb:1100]: lower case:
- Vertex AI hyperparameter tuning

**Review by kittyabs — COMMENTED** [review]
some needed edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**